Bearing/BearingChapterBuilder.cpp

```diff
@@ -51,15 +52,1161 @@ Uint16 CBearingChapterBuilder::GetMaxLevel() const
+	}
+	else
+	{
+		*pPara << symbol(RIGHT_SINGLE_ARROW) << n << _T(" > ") << n_max_stab_y << _T(" ") << RPT_PASS;
+	}
+	*pPara << _T(" (SECTION 14.7.6.3.6)");
+
+
+	pSubHeading = new rptParagraph(rptStyleManager::GetSubheadingStyle());
+	(*pChapter) << pSubHeading;
+	*pSubHeading << _T("Maximum Compressive Strain:");
+	pPara = new rptParagraph;
+	(*pChapter) << pPara;
+	*pPara << _T("Concrete Elastic Modulus, ") << Sub2(_T("E"), _T("c")) << _T(" = 3");
+	*pPara << symbol(TIMES) << Sub2(_T("(G"), _T("min")) << _T(" + ") << Sub2(_T("G"), _T("max")) << _T(")");
+	*pPara << symbol(TIMES) << _T("S") << Super(_T("2")) << _T(" = 3") << symbol(TIMES) << _T("(");
+	*pPara << E.SetValue(Gmin) << _T(" + ") << E.SetValue(Gmax) << _T(")") << symbol(TIMES);
+	*pPara << s << Super(_T("2")) << _T(" = ") << E.SetValue(EcA) << rptNewLine;
+	if (max_comp_strain_check == true)
+	{
+		*pPara << symbol(RIGHT_SINGLE_ARROW) << stress.SetValue(tl_stress) << _T("/");
+		*pPara << E.SetValue(EcA) << _T(" = ") << tl_stress / EcA << _T(" < 0.07 ") << RPT_PASS;
+	}
+	else
+	{
+		*pPara << symbol(RIGHT_SINGLE_ARROW) << stress.SetValue(tl_stress) << _T("/");
+		*pPara << E.SetValue(EcA) << _T(" = ") << tl_stress / EcA << _T(" > 0.07 ") << RPT_FAIL;
+	}
+	*pPara << _T(" (14.7.6.3.3-1)");
+
+	pSubHeading = new rptParagraph(rptStyleManager::GetSubheadingStyle());
+	(*pChapter) << pSubHeading;
+	*pSubHeading << _T("Initial Dead Load Deflection:");
+	pPara = new rptParagraph;
+	(*pChapter) << pPara;
+
+	*pPara << Sub2(symbol(delta), _T("DL-initial")) << _T(" = ") << Sub2(_T("P"), _T("DL")) << symbol(TIMES) << Sub2(_T("h"), _T("ri"));
+	*pPara << _T("/A/") << Sub2(_T("E"), _T("c")) << _T(" = ");
+	*pPara << force.SetValue(dl) << symbol(TIMES) << length.SetValue(total_elastomer_thickness);
+	*pPara << _T("/") << _T("/") << area.SetValue(a) << _T("/") << E.SetValue(EcA) << _T(" = ");
+	*pPara << length.SetValue(deltaDLiA) << _T(" (AASHTO Comm. C14.7.5.3.6-1)");
+
+
+	pSubHeading = new rptParagraph(rptStyleManager::GetSubheadingStyle());
+	(*pChapter) << pSubHeading;
+	*pSubHeading << _T("Instantaneous Live Load Deflection:");
+	pPara = new rptParagraph;
+	(*pChapter) << pPara;
+
+	*pPara << Sub2(symbol(delta), _T("LL")) << _T(" = ") << Sub2(_T("P"), _T("LL")) << symbol(TIMES) << Sub2(_T("h"), _T("ri"));
+	*pPara << _T("/A/") << Sub2(_T("E"), _T("c")) << _T(" = ");
+	*pPara << force.SetValue(ll) << symbol(TIMES) << length.SetValue(total_elastomer_thickness);
+	*pPara << _T("/") << _T("/") << area.SetValue(a) << _T("/") << E.SetValue(EcA) << _T(" = ");
+	*pPara << length.SetValue(deltaLLiA) << _T(" (AASHTO Comm. C14.7.5.3.6-1)");
+
+
+
+}
+
 rptChapter* CBearingChapterBuilder::Build(const std::shared_ptr<const WBFL::Reporting::ReportSpecification>& pRptSpec,Uint16 level) const
 {
-   rptChapter* pChapter = new rptChapter;
-   rptParagraph* pPara;
 
-   pPara = new rptParagraph;
+
+   CEAFApp* pApp = EAFGetApp();
```

Comment:
Lines 1185 - 1191 aren't needed in this function. The object created don't ever get used. This is left over code from when you refactored this method.

Reply:
these lines have been removed